concert/app/src/layer/python/enchanted_brain/attributes.py

```diff
@@ -1,5 +1,10 @@
 ATTR_RECORD_ID = "recordId"
 ATTR_CREATED_AT = "createdAt"
+ATTR_STAGE_ID = "stageId"
+ATTR_DISPLAY_NAME = "displayName"
+ATTR_END_TIME = "endTime"
+ATTR_SONGS = "songs"
```

Comment:
Please use prefixes (e.g. `ATTR_SONG_LIST_*`) for attributes that are only used for one record type.

Reply:
Done.